django/db/backends/mysql/features.py

```diff
@@ -48,6 +48,8 @@ class DatabaseFeatures(BaseDatabaseFeatures):
             SET V_I = P_I;
         END;
     """
+    # MySQL has TRADITIONAL instead of TEXT, but we alias TRADITIONAL to TEXT for consistency
```

Comment:
Chop "we" per our [style guide](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style).